Project-1/README.md

```diff
@@ -12,55 +12,303 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+  
+A B ¬A ¬B (A→B) (A→¬B) (A→B)∧(A→¬B)
+T T  F  F   T     F         F
+T F  F  T   F     T         F
+F T  T  F   T     T         T
+F F  T  T   T     T         T
+
+- If I am doing this right, this is a contingent.
+  
+  ```(c) (A→(B∨C))∨(C→¬A) 
+  
+ - (A→(B∨C)) means if 'A', then either 'B', 'C', or both.
+ - (C→¬A) means if 'C', then 'not A'.
+ - (A→(B∨C))∨(C→¬A) means either one or both of the above statements are true.
+ 
+A B C  A→(B∨C) (C→¬A) (A→(B∨C))∨(C→¬A)
+T T T     T       F            T
+T F T     T       F            T   
+T T F     T       F            T
+F T T     T       T            T
+F T F     T       T            T
+F F T     T       T            T
+T F F     F       T            T
+F F F     T       T            T
+
+- If I am doing this right, this is a tautology.
+ 
+  ```(d) ((A→B)∧C)∨(A∧D)
+  
+A B C D ((A→B)∧C)(A∧D) ((A→B)∧C)∨(A∧D)
+F F F F	    F.     F.          F                      
+F F F T	    F.     F.          F
+F F T F	    F.     T.          T
+F F T T	    T      F           T
+F T F F	    F.     F.          F
+F T F T	    F.     F.          F
+F T T F	    T      T.          T
+F T T T	    T.     F.          T
+T F F F	    F.     F.          F     
+T F F T	    T.     T.          T
+T F T F	    F.     F.          F
+T F T T	    F.     T.          T
+T T F F	    F.     F.          F
+T T F T	    F.     T.          T
+T T T F	    T.     F.          T
+T T T T     T.     T.          T
+  
+  - If I am correct, this is a contingent.
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
   ```(a) (A→B)→C
+
+A→B ≡ ¬A∨B
+
+(A→B)→C ≡ ¬(¬A∨B)∨C
+
+¬(¬A∨B)∨C ≡ (A∧¬B)∨C
+
+(A∧¬B)∨C ≡ (C∨A)∧(C∨¬B)
+
+CNF: (C∨A)∧(C∨¬B)
+
   (b) (A→(B∨C))∨(C→¬A)
+  
+A→(B∨C) ≡ ¬Av(B∨C)
+
+C→¬A ≡ ¬C∨¬A
+
+(A→(B∨C))∨(C→¬A) ≡ (¬Av(B∨C))v(¬C∨¬A)
+
+- Ali: ((¬AvB)v(¬AvC))v(¬C∨¬A)
+
+- Karl, Sydney, Ali: This is a tautology. Any tautology written in CNF will yield the same result and works as a conversion.
+
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
-```
+  
+- Plugging this statement into the Stanford Truth Table Generator shows that it is also a tautology.
+
+- So, both (2b) and (2c) can be rewritten as a CNF formula with no terms (got this from stack exchange, https://math.stackexchange.com/questions/64473/what-is-the-conjunction-normal-form-of-a-tautology), so 'T'. I've also seen A∨¬A used as a simplification.
+
 
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+
   ```(a)  B(x,y) that says that x is a brother of y
+  
+   B(x,y) ≡ ¬F(x)∧P(z,y)∧P(z,x)
```

Comment:
Counterexample: x=y

Also, this says that x is brother of y just in case x isn't female and everyone is a parent of y and everyone is a parent of x. You should use existential quantifiers to limit to scope of z.